addons/spreadsheet/static/src/chart/plugins/odoo_chart_core_plugin.js

```diff
@@ -186,6 +187,10 @@ export default class OdooChartCorePlugin extends CorePlugin {
      */
     _addOdooChart(chartId, fieldMatching = {}) {
         const charts = { ...this.charts };
+        if (JSON.stringify(fieldMatching) === "{}") {
```

Comment:
define the default fieldMatching value as undefined to not have tho JSONifiy the value